### DISCUSSION TITLE
Git should collect all SOLVED git user problems and store them for future user references

### DISCUSSION THREAD

**Discussion by joshjlgr** [community/community#8198] [Other Feature Feedback, Questions, & Ideas]
Very easy to run into technical issues and have to learn and fix them

**Comment by octacore4F-dev** [comment]
KEKW

**Reply by Sveta5** [reply]
****

**Comment by HonaiSport23** [comment]
Getting almost errors I can rectify the Rnning or any kind of Codes and commands and also I stallions of saoftwares.

**Comment by YVS37** [comment]
J

**Comment by Ffdjki** [comment]
- [ ]

**Comment by samus-aran** [comment]
We hope y'all found what you were looking for. ✨

We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
